server/routes/linkExistingAddressToChain.ts

```diff
@@ -18,6 +19,9 @@ const linkExistingAddressToChain = async (models, req: Request, res: Response, n
   if (!req.body.address) {
     return next(new Error(Errors.NeedAddress));
   }
+  if (!req.body.encodedAddress) {
```

Comment:
It looks like `req.body.address` is unused now -- why have two parameters?

Reply:
We use `req.body.address` to get the original address.
```
const originalAddress = await models.Address.scope('withPrivateData').findOne({
    where: { chain: req.body.originChain, address: req.body.address, user_id: userId, verified: { [Op.ne]: null } }
  });
```

Reply:
Previously, when user signs in to `edgeware` and joins to `centrifuge`, It copied the `edgeware` address and just changed the `chain`.
But we also need to change the address with `centrifuge` encoded address.

Reply:
Why not do the reencoding on the server side, since we have all the SS58Prefixes in the database?

Reply:
cool. that's a better idea. I was not sure if I can re-encode on the server side. (in other words, I was not sure if the polkadot/utils library was installed. I was lazy to check that. :) )

Reply:
done